include/ua_types.h

```diff
@@ -745,6 +745,8 @@ struct UA_DataType {
                                     pointers */
     UA_Boolean overlayable  : 1; /* The type has the identical memory layout in
                                     memory and on the binary stream. */
+    UA_UInt32  xmlEncodingId;    /* NodeId of datatype when encoded as XML */
```

Comment:
I used UInt32 because UA_NodeId also uses UInt32 for the numeric representation. I'll change it here to UInt16

What about the UInt32 in UA_NodeId? Should we also change it there?